### ISSUE TITLE
Translate flag "manager" in LDAP CustomerUser mapping to mail or SamAccountName

### ISSUE THREAD

**Issue by StefanRother-OTOBO** [RotherOSS/otobo#2557]
With an Active Directory or LDAP connection, the "manager" flag is increasingly required in the approval process. However, since the flag only provides the DN of the manager, we usually need Mail or sAMAccountName in processes or notifications, the option to translate the flag accordingly was added in the "CustomerUser" option.

For this, only a mapping for the flag must be configured 

`[ 'UserManager',       Translatable('Manager'),               'manager',     1, 0, 'var', '', 1, undef, undef ],`

and the new subconfiguration

```
# Translate manager flag in mapping to mail, uid, SamAccountName etc...
         TranslateManagerTo => 'sAMAccountName', 
```

be activated.

**Comment by StefanRother-OTOBO** [comment]
Enhancement added to OTOBO 11.

**Comment by bschmalhofer** [comment]
Set the milestone to 11.0.1 as this feature was released with OTOBO 11.0.1.
